hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/BackupSystemTable.java

```diff
@@ -1720,7 +1723,7 @@ public String[] getListOfBackupIdsFromMergeOperation() throws IOException {
       if (val.length == 0) {
         return null;
       }
-      return new String(val).split(",");
+      return new String(val, StandardCharsets.UTF_8).split(",");
```

Comment:
Ditto.

Reply:
It does not produce a warning, because stringsplitter warnings are suppressed.